extensions/npc/standalone.mjs

```diff
@@ -4,6 +4,16 @@
 import fs from 'fs';
 import InteractiveObjectServerBaseClass from '../../common/interactive-object/server.mjs';
 
+/**
+ * TODO: jsdoc
+ */
+class BulletinNPC extends InteractiveObjectServerBaseClass {
+  constructor(helper, objectName, configFilePath) {
+    super(helper, objectName, configFilePath);
+    this.editableDialogs = {"<editable1>": "Welcome to HITCON online bulletin"};
```

Comment:
I would recommend naming it `globalVars` instead of `editableDialogs`. This way, this object will be a generic feature for storing any state information for the NPC, instead of being specific to bulletin dialogs, and we'll have more reasons to make it part of the standard InteractiveObject code base.